WordPress/Classes/ViewRelated/System/Search/WPSearchControllerConfigurator.h

```diff
@@ -0,0 +1,17 @@
+#import <Foundation/Foundation.h>
+#import "WPSearchController.h"
+
+extern const CGFloat SearchBarWidth;
+extern const CGFloat SearchBariPadWidth;
+extern const CGFloat SearchWrapperViewPortraitHeight;
+extern const CGFloat SearchWrapperViewLandscapeHeight;
+extern const NSTimeInterval SearchBarAnimationDuration;
+
+@interface WPSearchControllerConfigurator : NSObject
```

Comment:
Hey @kwonye.  You should do a couple of things:

1. Move the documentation right above the `@interface` declaration.  Placing it on top of the constants makes it look as if it refers to them.
2. Make sure you're using doxygen style documentation (main keywords are `@class`, `@brief`, `@details`, `@param <paramName>` and `@returns`).
3. Specifically add a `@class` directive - I remember this directive was kind of necessary for documenting classes and getting Xcode to recognize the comment.  Not sure why.

As an example of how it should look like:

```objective-c
/**
 * @class WPSearchControllerConfigurator
 * @brief Worker class that configures the SearchController and SearchBar properties that multiple
 * view controllers have.
 * @details Once WPSearchController is deprecated (iOS 7 support) is removed, searchController
 * must be an instance of UISearchController
 */
```